eppic-cli/src/main/java/eppic/assembly/CrystalAssemblies.java

```diff
@@ -587,19 +587,36 @@ public void score() {
 			indx++;
 		}
 		
+		// This is done when multiple assemblies are high scoring
+		indx = 0;
+		List<Integer> indices = new ArrayList<Integer>();
+		for (Assembly a:uniques) {
+			if (a.getScore() == maxScore){
```

Comment:
Should I use some epsilon for the comparison, or do we trust score differences even if they are very tiny?